spec/person_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "./lib/person"
+RSpec.describe Person do
+
+  it "exists" do
+    person = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+    expect(person).to be_a(Person)
+    end
+
+  it "attributes" do
+    person = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+    expect(person.name).to eq('Hector')
+    expect(person.interests).to eq(["sewing", "millinery", "drawing"])
+    expect(person.supplies).to eq({})
+  end
+
+  it "add supply" do
+    person = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+    person.add_supply('fabric', 4)
+    person.add_supply('scissors', 1)
+    expect(person.supplies).to eq({"fabric"=>4, "scissors"=>1})
+    person.add_supply('fabric', 3)
+    expect(person.supplies).to eq({"fabric"=>7, "scissors"=>1})
+  end
+
+  it "can build?" do
+      hector = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+      sewing = Craft.new('sewing', {fabric: 5, scissors: 1, thread: 1, sewing_needles: 1})
+      expect(hector.can_build?(sewing)).to eq(false)
+      hector.add_supply('fabric', 7)
+      hector.add_supply('thread', 1)
+      expect(hector.can_build?(sewing)).to eq(false)
+      hector.add_supply('scissors', 1)
+      hector.add_supply('sewing_needles', 1)
+      expect(hector.can_build?(sewing)).to eq(true)
+  end
```

Comment:
I mentioned above that your logic was a bit off for this method. I don't think you were checking the actual supplies, just the quantities. A good way to test would be to try to add different supplies other than "scissors", "sewing_needles", or "fabric", or "thread". For example if you changed all of the supplies added to "dirt" I think the test would still pass.